app/models/user.rb

```diff
@@ -121,6 +130,18 @@ def superuser?; superuser end
 
   def developer?; developer end
 
+  def ambassador?
+    ambassador_index =
```

Comment:
Another benefit of the scope option means that this could be reduced to `memberships.ambassador_organizations.any?`